README.md

```diff
@@ -20,11 +20,21 @@ tables.
 The implementation is based on Spark [DataSources API][spark-ds-api] and Exasol
 [Sub Connections][sol-546].
 
+* [Prerequisite](#prerequisite)
 * [Quick Start](#quick-start)
 * [Usage](#usage)
 * [Configuration](#configuration)
 * [Building and Testing](#building-and-testing)
 
+## Prerequisite
+
+* Deployed and running Spark cluster
+* Deployed and running Exasol cluster
```

Comment:
I think a hint here for the number of executor condition would be great :)

Reply:
I added short description. Please have a look if it makes sense.

On the FAQ, there is more detail about it, if Spark cannot start more tasks than Exasol nodes.